.github/workflows/update_catalog.yml

```diff
@@ -0,0 +1,47 @@
+on:
+  push:
```

Comment:
This should limit deploys to updates to the `catalog.json` file on the `master` branch:

```suggestion
  push:
    branches:
      - master
    paths:
      - "Catalog/catalog.json"
```

Reply:
Just curious, why does this limit deploys?